docs/src/tutorials/eeg_animation.jl

```diff
@@ -0,0 +1,94 @@
+using Javis
+using Luxor
+
+function ground(args...)
+    background("white")
+    sethue("black")
+end
+
+function draw_line(p1 = O, p2 = O,  color = "black", action = :stroke, edge = "solid")
+    sethue(color)
+    setdash(edge)
+    line(p1, p2, action)
+end
+
+function circ(p = O, color = "black", action = :fill, radius = 25, edge = "solid")
+    sethue(color)
+    setdash(edge)
+    circle(p, radius, action)
+    return Transformation(p, 0.0)
+end
+
+radius = 15
+demo = Video(500, 500)
+javis(
+    demo,
+    [
+        Action(1:100, ground),
+        Action(
+            :same,
+            :inside_circle,
+            (args...) -> circ(O, "black", :stroke, 140, "longdashed"),
+        ),
+        Action(:same, :head, (args...) -> circ(O, "black", :stroke, 170)),
+
+        Action(:same, :vert_line, (args...) -> draw_line(Point(0, -170), Point(0, 170),"black", :stroke, "longdashed")),
+        Action(:same, :horiz_line, (args...) -> draw_line(Point(-170, 0), Point(170, 0),"black", :stroke, "longdashed")),
+
+        Action(:same, :cz, (args...) -> circ(O, "white", :fill, radius)),
+        Action(:same, :c3, (args...) -> circ(Point(-70, 0), "white", :fill, radius)),
+        Action(:same, :c4, (args...) -> circ(Point(70, 0), "white", :fill, radius)),
```

Comment:
You use the code quite often and just change the position if I see this correctly. Why not specify a vector of points instead and use `circ.(points, ....)` :wink: